game/effect.go

```diff
@@ -0,0 +1,10 @@
+package game
+
+import ()
+
+type Effect struct {
+	BasePower     int
```

Comment:
this seems like it is "Base" in no way, might as well just call it Power and Toughness